features/step_definitions/networking.rb

```diff
@@ -989,3 +989,13 @@
   raise "OVN is not running correctly! Check one of your ovnkube-node pod" unless desired_ovnkube_node_replicas == available_ovnkube_node_replicas && available_ovnkube_node_replicas != 0
   raise "OVN is not running correctly! Check one of your ovnkube-master pod" unless desired_ovnkube_master_replicas == available_ovnkube_master_replicas && available_ovnkube_master_replicas != 0
 end
+
+Given /^I get the ptp logs of the "([^"]*)"  node since "(.+)" ago$/ do | node_name, duration |
```

Comment:
can you please add a comment about what unit `duration` should be?

Reply:
is the an extra space between `"([^"]*)"  node`?

Reply:
Thanks, comments added and extra space removed.